externals/kyuubi-spark-sql-engine/src/main/resources/python/execute_python.py

```diff
@@ -15,28 +15,34 @@
 # limitations under the License.
 #
 
-from glob import glob
 import ast
-import sys
 import io
 import json
-import traceback
-import re
+import kyuubi_util
```

Comment:
Since the `kyuubi_util` import `py4j` module, we can't call `import kyuubi_util` before adding the `py4j` module to our `sys.path`, in another word, we can't call it before L223.

```
Traceback (most recent call last):
  File "/tmp/execute_python.py", line 21, in <module>
    import kyuubi_util
  File "/private/tmp/kyuubi_util.py", line 20, in <module>
    from py4j.clientserver import ClientServer, JavaParameters, PythonParameters
ModuleNotFoundError: No module named 'py4j'
```

Reply:
Thanks for the hint and it works.
Is it a reverse dependency pattern here? shall we move the patch py4j path to the top .

Reply:
SGTM